Library/Homebrew/cmd/update-bash.sh

```diff
@@ -305,7 +323,7 @@ EOS
   wait
   trap - SIGINT
 
-  for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
+  for DIR in "${DIRS[@]}"
   do
     [[ -d "$DIR/.git" ]] || continue
     pull "$DIR"
```

Comment:
If you reset, then what's the point to run `pull` which will only create unnecessary IO overhead? I'm missing something that `pull` will perform not covered by `reset`?